bedrock/firefox/templates/firefox/browsers/chromebook.html

```diff
@@ -0,0 +1,92 @@
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ ftl('browsers-chromebook-get-firefox-browser') }}{% endblock %}
+
+{% block page_desc %} {{ ftl('browsers-chromebook-so-youve-got') }} {% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('chromebook') }}
+{% endblock %}
+
+{% block content %}
+<div class="mzp-l-content">
+  {% call hero(
+    title=ftl('browsers-chromebook-get-firefox-browser'),
+    class='mzp-has-image mzp-t-firefox mzp-t-product-firefox',
+    image_url='img/firefox/browsers/quantum/browser.jpg',
+    heading_level=1
+  ) %}
+  <p>{{ ftl('browsers-chromebook-so-youve-got') }}</p>
+
+  <div class="mzp-c-menu-list mzp-t-cta mzp-t-download" id="chromebook-download-list">
+    <h4 class="mzp-c-menu-list-title">{{ ftl('browsers-chromebook-dropdown-copy') }}</h4>
+    <ul class="mzp-c-menu-list-list download-platform-list">
+      <li class="mzp-c-menu-list-item">
+        <a href="{{ firefox_adjust_url('android', 'firefox-browsers-faq') }}" rel="external noopener" data-link-type="download" data-display-name="Android" data-download-version="android" data-download-os="Android">
+          {{ ftl('browsers-chromebook-get-firefox-for') }}
+          <p><small>{{ ftl('browsers-chromebook-x86-based-chromebook') }}</small></p>
+        </a>
+      </li>
+      <li class="mzp-c-menu-list-item t-sumo">
+        <a href="https://support.mozilla.org/kb/run-firefox-chromeos?utm_source=www.mozilla.org-firefox-browsers-chromebook&amp;utm_medium=referral&amp;utm_campaign=seo" rel="external noopener" data-cta-text="Install Firefox for Chromebook" data-cta-type="link">
+          {{ ftl('browsers-chromebook-get-firefox-desktop') }}
+        </a>
+      </li>
+    </ul>
+  </div>
+
+  {% endcall %}
+
+  <section>
+    <h2>{{ ftl('browsers-chromebook-why-get-firefox') }}</h2>
+    {% call feature_card(
+      image_url='img/firefox/browsers/chromebook/shield.png',
+      include_highres_image=True,
+      aspect_ratio='mzp-has-aspect-1-1',
+      class='mzp-l-card-feature-left-half mzp-t-firefox'
+    ) %}
+
+      <p>{{ ftl('browsers-chromebook-while-a-chromebook') }}</p>
+      <ul class="mzp-u-list-styled">
+        <li>{{ ftl('browsers-chromebook-alwayson-tracking-protection') }}</li>
+        <li>{{ ftl('browsers-chromebook-supporting-independent-technology') }}</li>
+      </ul>
+    {% endcall %}
+  </section>
+
+  <section>
+    <div class="mzp-l-content mzp-t-content-md">
+      <h2>{{ ftl('browsers-chromebook-is-it-hard') }}</h2>
+      <p>{{ ftl('browsers-chromebook-we-wish-it') }}</p>
+    </div>
+
+    <div class="mzp-l-content mzp-l-card-half">
+      <div class="mzp-c-card">
+        {{ high_res_img('img/firefox/browsers/chromebook/mobile.png', {'alt': '', 'class': 'mzp-c-billboard-image', 'width': '436', 'height': '464'}) }}
+
+        <p>{{ ftl('browsers-chromebook-install-firefox-from') }}</p>
+        {{ google_play_button(href=android_url, id='play-store-link') }}
```

Comment:
You need to define android_url (I keep making this mistake myself 😆 )

`{% set android_url = firefox_adjust_url('android', 'firefox-daylight-promo-banner') %}`